examples/counter-next/shared/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "shared"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.80"
+
+[lib]
+crate-type = ["lib", "staticlib", "cdylib"]
+name = "shared"
+
+[[bin]]
+name = "crux_cli"
+required-features = ["cli"]
```

Comment:
Does this make the building of the bin target conditional?

Reply:
It means we have to build it with the `cli` feature, which, in turn, brings in the "cli" feature from `crux_core`, which in turn re-exports from `crux_cli` 😓

so:

```bash
cargo run --package shared --bin crux_cli --features cli -- --help
```